katharsis-core/src/main/java/io/katharsis/resource/registry/responseRepository/RelationshipRepositoryAdapter.java

```diff
@@ -1,12 +1,23 @@
 package io.katharsis.resource.registry.responseRepository;
 
+import java.io.Serializable;
+import java.util.Set;
+
 import io.katharsis.queryParams.QueryParams;
+import io.katharsis.queryspec.DefaultQuerySpecConverter;
+import io.katharsis.queryspec.FilterOperator;
+import io.katharsis.queryspec.FilterOperatorRegistry;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.queryspec.FilterOperatorRegistry'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)